src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -1074,7 +1074,7 @@
     <Compile Include="$(CommonPath)Interop\Interop.TimeZoneDisplayNameType.cs">
       <Link>Common\Interop\Interop.TimeZoneDisplayNameType.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)Interop\Interop.TimeZoneInfo.cs" Condition="'$(TargetsBrowser)' != 'true'">
+    <Compile Include="$(CommonPath)Interop\Interop.TimeZoneInfo.cs">
```

Comment:
Because we aren't using `TimeZoneInfo.GetDisplayName.Invariant.cs` anymore (will also remove that file in this PR) so `Interop.Globalization` is not defined for Browser